requirements.txt

```diff
@@ -1,3 +1,4 @@
 aiohttp
 cloudflare>=2.3.1
+Numpy
```

Comment:
Numpy is a 20MB  library.. Do we have a real use-case for importing this?

Reply:
it's use from percentille. do you know a smaller lib for that ?

Reply:
The stdlib? https://docs.python.org/3/library/statistics.html

Reply:
quantiles ? -> new in 3.8

Reply:
OK, fair point. Still I'd require an advice from someone on #python for ex, it's really worth avoiding 20MB libs just for the purpose of doing quantiles..